### DISCUSSION TITLE
Flugersatzverkehr

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#55] [Ideas]
Dazu bräuchte man Flughafenbahnhöfe, die dann direkt verbunden werden sollen.

Die Direktausschreibungen wären dann besonders gut für lange Laufwege und V60- und n-Wagen-Garnituren geeignet.

**Comment by marhei** [comment]
Es gibt jetzt noch einen Flughafen… 👀

**Comment by marhei** [comment]
Und noch einer! 👀
